src/modules/todos.js

```diff
@@ -0,0 +1,100 @@
+import CompletedTasksStatus from './tasksCompleted.js';
+
+const status = new CompletedTasksStatus();
+export default class Tasks {
+  constructor() {
+    this.tasksArray = JSON.parse(localStorage.getItem('tasks')) || [];
+  }
+
+  saveAndRender = () => {
+    localStorage.setItem('tasks', JSON.stringify(this.tasksArray));
+    this.displayList();
+  }
+
+  displayList = () => {
+    // display tasks
+    const todoContainer = document.querySelector('#todo-list');
+    todoContainer.innerHTML = '';
+    this.tasksArray.forEach((task) => {
+      const li = document.createElement('li');
+      li.className = 'todo-task';
+      li.id = task.index;
+      li.innerHTML = `<div><button class="check-task"><i class="fa-regular fa-square"></i><i class="fa-solid fa-check"></i></button> <input class="todo-input" type="text" value="${task.description}"></div><button><i class="fa-solid fa-ellipsis-vertical" style="margin-left:15rem;"></i>
+      </button><button class="delete-task"><i class="fa-solid fa-trash-can"></i></button>`;
+      todoContainer.insertBefore(li, todoContainer.children[task.index]);
+      if (task.isCompleted) {
+        li.classList.add('active');
+      }
+    });
+
+    // delete item
+    const deleteBtn = document.querySelectorAll('.delete-task');
+    deleteBtn.forEach((button, index) => {
+      button.addEventListener('click', () => {
+        this.remove(index);
+      });
+    });
```

Comment:
- This feature was there on the project output, however `this.remove(index)` will remove only the child elements i.e the empty list item was still displaying on the UI, so replace `this.remove(index)` with `this.parentElement.remove(index)` to remove li tag which is the parent element.

Reply:
Thanks @gergitokusse, for your constructive feedback 👍 I really appreciate that 💯